<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Contacts</class>
 <widget class="QWidget" name="Contacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <widget class="QCheckBox" name="mainStatusShow">
        <property name="text">
         <string>Show contact status text in contact list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainMoodShow">
        <property name="text">
         <string>Show mood icon in contact list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainGeneralActivityShow">
        <property name="text">
         <string>Show main activity icon in contact list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainExtendedActivityShow">
        <property name="text">
         <string>Show extended activity icon in contact list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainTuneShow">
        <property name="text">
         <string>Show tune icon in contact list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainAuthShow">
        <property name="text">
         <string>Show not authorized icon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainQIPStatusShow">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show QIP xStatus in contact list</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mainResNotifyShow">
        <property name="text">
         <string>Show main resource in notifications</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>mainGeneralActivityShow</sender>
   <signal>clicked(bool)</signal>
   <receiver>mainExtendedActivityShow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
